plugin/src/main/groovy/com/btkelly/gnag/tasks/DetektTask.groovy

```diff
@@ -41,6 +45,11 @@ final class DetektTask {
                 args "--config", "$reporterConfig"
             }
         }
+
+        result.logging.captureStandardOutput(STD_OUT_LOG_LEVEL)
+        result.logging.captureStandardError(STD_ERR_LOG_LEVEL)
```

Comment:
For some reason the project-level captures don't seem to work for this task, so I had to re-specify them here.